apps/mark-scan/frontend/src/components/accessible_controller_illustration.tsx

```diff
@@ -0,0 +1,240 @@
+export type ControllerButton =
+  | 'up'
+  | 'down'
+  | 'left'
+  | 'right'
+  | 'select'
+  | 'rate-up'
+  | 'rate-down'
+  | 'volume-up'
+  | 'volume-down'
+  | 'help'
+  | 'pause';
+
+interface AccessibleControllerIllustrationProps {
+  highlight?: ControllerButton;
+}
+
+export function AccessibleControllerIllustration({
+  highlight,
+}: AccessibleControllerIllustrationProps): JSX.Element {
+  const highlightFill = '#985aa3';
```

Comment:
What theme color specifically? I don't know if there's a theme color that suits this application well.

Reply:
Was thinking that since this looks like it's using a version of the old vxpurple, `colors.primary` is the closest equivalent to use here - and then the dark text/line/fill colors would be `colors.onBackground` to match the app text color

Reply:
I didn't want to use `colors.primary` because, as a background for a button with an icon, it kind of obscures the icon:

<img width="419" alt="Screen Shot 2024-03-18 at 11 42 51 AM" src="https://github.com/votingworks/vxsuite/assets/37960853/5447cf27-d863-4f83-93eb-b4ba287bb8a6">

To at least get within the new VxPurple range, I dropped in the `Purple60` from the desktop theme:

<img width="435" alt="Screen Shot 2024-03-18 at 11 46 07 AM" src="https://github.com/votingworks/vxsuite/assets/37960853/1f9730e1-eb82-4df1-802e-775486558905">